.github/workflows/tests.yml

```diff
@@ -1202,12 +1202,12 @@ jobs:
 
   valkey:
     env:
-      TOTAL_GROUPS: 2
+      TOTAL_GROUPS: 1
```

Comment:
Along with this change I think we also will need to edit the valkey_settings in db_settings to only return the first group too.

Reply:
This doesn't change the number of valkey "instances" available, it's just reducing the number of GHA runners used to run these tests. db_settings doesn't need to change.